ui/src/app/utils/stores.tsx

```diff
@@ -1,6 +1,6 @@
 import { BreadcrumbType } from 'app/utils/navigation';
 import {atom, Atom} from 'app/utils/subscribable';
-import {Profile} from 'generated';
+import {Profile} from 'generated/fetch';
```

Comment:
nit: collapse into line below (think this should be covered by lint)

Reply:
FYI: lint didn't catch - but fixed manually